jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/errorlist/ExecutionErrorListPresenter.java

```diff
@@ -243,32 +231,27 @@ public Menus getMenus() {
                 .build();
     }
 
-    protected List<ExecutionErrorSummary> getDisplayedExecutionErrors() {
-        return visibleExecutionErrors;
-    }
-
     public void selectExecutionError(final ExecutionErrorSummary summary,
                                      final Boolean close) {
         PlaceStatus status = placeManager.getStatus(new DefaultPlaceRequest(ExecutionErrorDetailsPresenter.SCREEN_ID));
         if (status == PlaceStatus.CLOSE) {
             placeManager.goTo(ExecutionErrorDetailsPresenter.SCREEN_ID);
-            execErrorSelectionEvent.fire(new ExecErrorSelectionEvent(getSelectedServerTemplate(),
-                                                                     summary.getDeploymentId(),
-                                                                     summary.getErrorId()));
+            executionErrorSelectedEvent.fire(new ExecutionErrorSelectedEvent(getSelectedServerTemplate(),
+                                                                             summary.getDeploymentId(),
+                                                                             summary.getErrorId(),
+                                                                             getErrorDetailTitle(summary)));
         } else if (status == PlaceStatus.OPEN && !close) {
-            execErrorSelectionEvent.fire(new ExecErrorSelectionEvent(getSelectedServerTemplate(),
-                                                                     summary.getDeploymentId(),
-                                                                     summary.getErrorId()));
+            executionErrorSelectedEvent.fire(new ExecutionErrorSelectedEvent(getSelectedServerTemplate(),
+                                                                             summary.getDeploymentId(),
+                                                                             summary.getErrorId(),
+                                                                             getErrorDetailTitle(summary)));
         } else if (status == PlaceStatus.OPEN && close) {
             placeManager.closePlace(ExecutionErrorDetailsPresenter.SCREEN_ID);
         }
     }
 
-    public void onExecutionErrorSelectionEvent(@Observes ExecErrorWithDetailsRequestEvent event) {
-        placeManager.goTo(ExecutionErrorDetailsPresenter.SCREEN_ID);
-        execErrorSelectionEvent.fire(new ExecErrorSelectionEvent(event.getServerTemplateId(),
-                                                                 event.getDeploymentId(),
-                                                                 event.getErrorId()));
+    private String getErrorDetailTitle(ExecutionErrorSummary summary){
```

Comment:
@nmirasch I think this should be done in the presenter that receives the event (`ExecutionErrorDetailsPresenter`). That presenter is the one responsible for building the UI data, the event would only have the necessary ids that identify the error that was selected.